src/main/java/com/google/sps/servlets/GoodDeedsServlet.java

```diff
@@ -80,15 +83,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect(REDIRECT_HOMEPAGE);
     }
  
-    private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    public String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
Are these public for testing? Noticed that the tests are in a different package. Perhaps move the test to package com.google.sps.data.servlets and make the methods under test package-private.